screen/list_table.go

```diff
@@ -48,19 +37,58 @@ func ListTable(th *material.Theme, state *state.State) Screen {
 				}),
 				layout.Stacked(rowInset(func(gtx layout.Context) layout.Dimensions {
 					return layout.Flex{}.Layout(gtx,
-						layout.Rigid(rowInset(material.Body1(th, fmt.Sprintf("%s %s", student.Surname, student.Name)).Layout)),
+						layout.Rigid(rowInset(material.Body1(th, fmt.Sprintf("%s %s  ", student.Surname, student.Name)).Layout)),
 						layout.Rigid(material.Button(th, &delete[index], "Delete").Layout),
 					)
 				})),
+				layout.Stacked(rowInset(func(gtx layout.Context) layout.Dimensions {
+					return layout.Flex{}.Layout(gtx,
+						layout.Rigid(rowInset(material.Body1(th, fmt.Sprintf("%s %s                   ", student.Surname, student.Name)).Layout)),
+						layout.Rigid(material.Button(th, &edit[index], "Edit").Layout),
+					)
+				})),
 			)
 		})
 	}
+}
+
+// ListStudent defines a screen layout for listing existing students.
+func ListTable(th *material.Theme, state *state.State) Screen {
+	var close widget.Clickable
+	list := widget.List{List: layout.List{Axis: layout.Vertical}}
+
+	students, err := state.Students()
+	if err != nil {
+		// TODO: Show user an error toast.
+		log.Printf("failed to fetch students: %v", err)
+		return nil
+	}
+	// var maxlength storage.StudentEntry                // DELETE AFTER !!!
+	// for _, i := range students {
+	//	   fmt.Println("i:", i)
+	// }
+	delete := make([]widget.Clickable, len(students))
+	edit := make([]widget.Clickable, len(students))
+
+	studentsLayout := generateStudentsList(th, list, students, delete, edit)
 
 	return func(gtx layout.Context) (Screen, layout.Dimensions) {
 		d := layout.Flex{Axis: layout.Vertical}.Layout(gtx,
 			layout.Flexed(1, rowInset(studentsLayout)),
 			layout.Rigid(rowInset(material.Button(th, &close, "Close").Layout)),
 		)
+		for i, v := range delete { // DOESN'T WORK !!!
```

Comment:
This is not gonna work. You need the same values of clickables, not their copies. 
Use this instead 
```go
for i := range delete { ... }
```